ansible/roles/icinga/defaults/main.yml

```diff
@@ -111,9 +111,19 @@ icinga__additional_groups:
 # .. envvar:: icinga__fqdn [[[
 #
 # The Fully Qualified Domain Name of this Icinga 2 node. This variable is used
-# during node registration in Icinga 2 Director and in the zone configuration.
+# during the node registration in the Icinga 2 Director and in the zone
+# configuration.
 icinga__fqdn: '{{ ansible_fqdn }}'
 
+                                                                   # ]]]
+# .. envvar:: icinga__display_name [[[
+#
+# The short display name of this Icinga 2 node. This variable is used during
+# the node registration in the Icinga 2 Director and in the zone configuration.
+icinga__display_name: '{{ (inventory_hostname_short | d(inventory_hostname.split(".")[0]))
+                          if (inventory_hostname_short | d(inventory_hostname.split(".")[0]) != "localhost")
+                          else ansible_hostname }}'
```

Comment:
I prefer the FQDN for the display_name, but this is still fine with me as you made it configurable.

Reply:
Yeah, I did it like this because the FQDN is already in the `hostname` variable (JSON parameter `object_name`)...and because it seems to match the most common approach in the Icinga docs....but as you say, it's configurable, so I think it can be left as-is